fabric/src/main/java/com/noxcrew/noxesium/mixin/performance/render/GuiHookMixin.java

```diff
@@ -110,4 +112,14 @@ public void renderTabList(GuiGraphics graphics, float partialTicks, CallbackInfo
     public void renderSavingIndicator(GuiGraphics graphics, float partialTicks, CallbackInfo ci) {
         // TODO implement optimizations
     }
+
+    @WrapWithCondition(method = "*", at = @At(value = "INVOKE", target = "Lnet/minecraft/util/profiling/ProfilerFiller;push(Ljava/lang/String;)V"), require = 0)
+    private boolean ignoreProfilerPush(ProfilerFiller profilerFiller, String name) {
```

Comment:
I would keep the profiler functional, previously it was kinda unintentionally removed as I had just replaced the whole method, but now with the new approach I dropped it. There's also no reason to remove them for any performance gains, and if anything any of the vanilla ones should just stay.